.github/workflows/ci.yaml

```diff
@@ -0,0 +1,55 @@
+name: ci
+
+# Build and run all tests.
+# - Triggers: All pushes and pull requests on any branch.
+# - Runs on: All repositories (including forks)
+
+on:
+  push:
+    branches: ['**']
```

Comment:
Isn't this make this run both builds (from ci and publish workflows) when a commit happens to main?

Reply:
Well noticed!

Suggestions?

A quick search tells me: that

```yaml
on:
  push:
    branches-ignore:
      - master
```

will do  the job.

Reply:
Nope! This will exclude all `master` branches, including those from forks.

I'll add a restriction to the job's if condition.